src/lib/components/index.tsx

```diff
@@ -14,12 +14,12 @@ const queryClient = new QueryClient({
   },
 })
 
-const MemoizedOverlay = memo(Overlay)
-
-export default function Index(props: any) {
-  return (
-    <QueryClientProvider client={queryClient}>
-      <MemoizedOverlay {...props} />
-    </QueryClientProvider>
-  )
+export default function ReactChatGPT(props: any) {
+  const { mode, host, ...rest } = props
+  if (host) {
+    console.log('host', host)
+    setHost(host)
+  }
+  return <QueryClientProvider client={queryClient}>{mode === 'auto' ? <ReactChatGPTIntegration {...rest} /> : <MemoizedOverlay {...rest} />}</QueryClientProvider>
```

Comment:
You may want to write instructions in the README to comment out the usage of the `mode` option.